src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/Collections.generated.txt

```diff
@@ -68,9 +74,41 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
 
             foreach (IConfigurationSection section in configuration.GetChildren())
             {
-                if (section.Value is string stringValue3)
+                if (section.Value is string value)
                 {
-                    obj.Add(stringValue3!);
+                    obj.Add(value);
+                }
+            }
+        }
+
+        public static void BindCore(IConfiguration configuration, ref List<int> obj, BinderOptions? binderOptions)
```

Comment:
>ref [](http://example.com/codeflow?start=66&length=3)

why this is a ref? Is it possible we create this list in other situations?

Reply:
Similar question here https://github.com/dotnet/runtime/pull/88338#discussion_r1260227905; I need to take a deeper look outside this PR.